src/screens/StartHardwareWallet.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag'
+
+import logo from '../assets/images/hathor-logo.png';
+import wallet from '../utils/wallet';
+import ledger from '../utils/ledger';
+import version from '../utils/version';
+import helpers from '../utils/helpers';
+import { LEDGER_GUIDE_URL, IPC_RENDERER, LEDGER_MIN_VERSION, LEDGER_MAX_VERSION } from '../constants';
+import InitialImages from '../components/InitialImages';
+import hathorLib from '@hathor/wallet-lib';
+
+
+/**
+ * Screen used to select between hardware wallet or software wallet
+ *
+ * @memberof Screens
+ */
+class WalletType extends React.Component {
```

Comment:
Should we call this class `StartHardwareWallet`?

Reply:
Yes! Just fixed it.